kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -9,69 +9,55 @@
       <h5 v-if="selectPracticeQuiz" class="select-folder-style">
         {{ selectPracticeQuizLabel$() }}
       </h5>
-      <KGrid v-else>
-        <KGridItem
-          :layout12="{ span: 6 }"
-          :layout8="{ span: 4 }"
-          :layout4="{ span: 2 }"
-        >
-          <h5 class="select-folder-style">
-            {{ selectResourcesDescription$() }}
-          </h5>
-          <span>
-            {{
-              maxNumberOfQuestionsPerSection$({
-                count: MAX_QUESTIONS_PER_QUIZ_SECTION, current: activeQuestions.length
-              })
-            }}
-          </span>
-        </KGridItem>
-        <KGridItem
-          :layout12="{ span: 6 }"
-          :layout8="{ span: 4 }"
-          :layout4="{ span: 2 }"
-        >
-          <div class="number-question">
-            <div>
-              <KTextbox
-                ref="numQuest"
-                v-model="questionCount"
-                type="number"
-                :label="numberOfQuestionsLabel$()"
-                :max="maxQuestions"
-                :min="1"
-                :invalid="questionCount > maxQuestions"
-                :invalidText="maxNumberOfQuestions$({ count: maxQuestions })"
-                :showInvalidText="true"
-              />
-            </div>
-            <div>
-              <div
-                :style="borderStyle"
-                class="group-button-border"
-              >
-                <KIconButton
-                  icon="minus"
-                  aria-hidden="true"
-                  class="number-btn"
-                  :disabled="questionCount === 1"
-                  @click="questionCount -= 1"
-                />
-                <span
-                  :style="dividerStyle"
-                > | </span>
-                <KIconButton
-                  icon="plus"
-                  aria-hidden="true"
-                  class="number-btn"
-                  :disabled="questionCount >= maxQuestions"
-                  @click="questionCount += 1"
-                />
-              </div>
-            </div>
+      <h5 class="select-folder-style">
```

Comment:
Regardless of whether it's in a KGridItem or not, it still needs to be in a v-else block, as this stuff isn't relevant for the 'Select Quiz' workflow.